glass/observations.py

```diff
@@ -101,7 +107,7 @@ def gaussian_nz(
     ----------
     z : array_like
         Redshift values of the distribution.
-    mode : float or array_like
+    mean : float or array_like
```

Comment:
It's the mode, not the mean!

Reply:
Should the argument name in function be changed from `mean` to `mode`?

Reply:
My mistake! I thought this was the other model (from which I guess the doctring was copied. `mean` is right!